services/frontend-service/src/ui/components/LockDisplay/LockDisplay.tsx

```diff
@@ -0,0 +1,68 @@
+/*This file is part of kuberpult.
+
+Kuberpult is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+Kuberpult is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with kuberpult.  If not, see <http://www.gnu.org/licenses/>.
+
+Copyright 2021 freiheit.com*/
+import { Button } from '../button';
+import { Delete } from '../../../images';
+import { DisplayLock } from '../../../api/api';
+
+export const LockDisplay: React.FC<{ lock: DisplayLock }> = (props) => {
+    const { lock } = props;
+    return (
+        <div className="lock-display">
+            <div className="-lock-display__table">
+                <div className="lock-display-table">
+                    <div
+                        className={
+                            'lock-display-info date-display--' + (isOutdated(lock.date) ? 'outdated' : 'normal')
+                        }>
+                        {daysToString(calcLockAge(lock.date))}
+                    </div>
+                    <div className="lock-display-info">{lock.environment}</div>
+                    {lock.application !== undefined && <div className="lock-display-info">{lock.application}</div>}
```

Comment:
```suggestion
                    { !! lock.application && <div className="lock-display-info">{lock.application}</div>}
```
is there a reason to use !== undefined ?